app/models/spree/calculator/weight/package_builder.rb

```diff
@@ -0,0 +1,134 @@
+module Spree
+  module Calculator::Weight
+    class PackageBuilder
+      extend Forwardable
+
+      attr_reader :shipping_calculator
+
+      def_instance_delegator :shipping_calculator, :max_weight_for_country
```

Comment:
Is there any reason why we can't initialise the `PackageBuilder` with the `max_weight_for_county` so it doesn't have to be coupled with the calculator?